### ISSUE TITLE
[dynamo] RNG seeding (Generator.seed + manual_seed) (1 models, 12 breaks)

### ISSUE THREAD

**Issue by penguinwu** [penguinwu/oss-model-graph-break-corpus#26]
## Summary

1 models produce 12 graph breaks from this pattern.

## Break Reason

```
Graph Break Reason: Encountered graph break when attempting to trace CALL: a function call, e.g. f(x, y):
```

## Affected Models

| Model | Modes | Breaks |
|-------|-------|--------|
| ReformerModel | train | 12 |

## Related Issues

- #10
- #54
- #56

## Tested On

- PyTorch 2.13.0.dev20260425+cu126
- Transformers 5.6.2
- Sweep date: 2026-04-26T08:48:25

**Comment by penguinwu** [comment]
### Nightly Validation (torch 2.12.0.dev20260407 vs 2.11.0 stable)

**Status: Persists.** ReformerModel still `graph_break` on v2.11 and nightly (eval and train).

**Conclusion: No change between v2.11 and nightly.** RNG seeding intentionally blocked by Dynamo.

**Comment by penguinwu** [comment]
## Sweep update — 2026-04-28

Verified on: `torch 2.13.0a0+gitf8d66d2` (source build, includes PR #181552) · transformers `5.6.2` · `fullgraph=True, dynamic=True`

**Status: Still firing (1/1 models)** — no change from PR #181552

**Minimal repro** (`ReformerModel`):
```bash
# In ~/projects/oss-model-graph-break-corpus, with venv activated:
echo '["ReformerModel"]' > /tmp/repro.json
python3 sweep/run_sweep.py sweep \
  --models /tmp/repro.json \
  --modes eval \
  --compile-kwargs '{"fullgraph": true, "dynamic": true, "backend": "eager"}' \
  --setup-script sweep/configs/animesh-logging-suppress.py
```

Break reason: `Data-dependent branching`

**Comment by penguinwu** [comment]
## Auto-closed by 2026-05-09 nightly sweep

Affected model now compiles fullgraph:
- `MraModel` (train) — full_graph

The gb8985 / gb7324 `requires_grad_() intermediate leaked as output` gap at `transformers/models/mra/modeling_mra.py:364` appears fixed in PyTorch 2.13.0.dev20260507+cu126. If incorrect, reopen with test demonstrating the gap.

<sub>Auto-closed via 2026-05-09 nightly sweep close-stale check.</sub>

**Comment by penguinwu** [comment]
## Reopened — closure was procedurally incorrect

Closed earlier today (2026-05-10 ~10:45 ET) via a one-off close-stale script
that bypassed the proper closure workflow (file-issue subagent + adversary
review + MRE re-verification of the gap on current torch). Reopening to
re-process through the canonical Step 2 workflow once that workflow is
properly defined.

The data evidence — that all affected models in this issue currently show
`status=full_graph` in `sweep_results/nightly/2026-05-09/identify_results.json`
— remains valid. But the causal claim ("torch fixed the gap") was not
verified per R3 (re-run the MRE on current torch nightly). Will re-evaluate
through the proper workflow.

<sub>Reopened 2026-05-10 ~11:10 ET. Per Peng directive: closures must follow
the gated file-issue close workflow, not one-off scripts.</sub>
